tcms/search/__init__.py

```diff
@@ -32,12 +32,7 @@ def fmt_queries(*queries):
     results = {}
     for _query in queries:
         for key, value in _query.items():
-            key = key.replace('p_product', 'product')
-            key = key.replace('p_', 'product ')
-            key = key.replace('cs_', 'case ')
-            key = key.replace('pl_', 'plan ')
-            key = key.replace('r_', 'run ')
-            key = key.replace('_', ' ')
+            key = get_refractored_key(key)
```

Comment:
Rename to `replace_keys()` or something closer to what this function actually does.